packages/amplify-e2e-tests/src/__tests__/auth_2.test.ts

```diff
@@ -1,33 +1,26 @@
 import {
   initJSProjectWithProfile,
   deleteProject,
-  amplifyPushAuth,
   amplifyPush,
   validateNodeModulesDirRemoval,
   updateFunction,
   addAuthwithUserPoolGroupsViaAPIWithTrigger,
 } from 'amplify-e2e-core';
-import { addAuthWithDefaultSocial, addAuthWithGroupTrigger, addAuthWithRecaptchaTrigger, addAuthViaAPIWithTrigger } from 'amplify-e2e-core';
+import { addAuthViaAPIWithTrigger } from 'amplify-e2e-core';
 import {
   createNewProjectDir,
   deleteProjectDir,
   getProjectMeta,
   getUserPool,
   getUserPoolClients,
-  isDeploymentSecretForEnvExists,
   getLambdaFunction,
-  removeAuthWithDefault,
 } from 'amplify-e2e-core';
 
 const defaultsSettings = {
   name: 'authTest',
 };
 
-describe('placeholder', () => {
```

Comment:
Removing this, which we'd disabled during migration, and getting rid of tests that don't interact w/ API.